Sources/Shared/ImageManager.swift

```diff
@@ -20,14 +20,14 @@ public class ImageManager {
   ///   - url: The URL of the image that should be fetched, either from network or cache (if cache is enabled).
   ///   - useCache: Determines if the image should be fetched from cache and stored in cached after fetching.
   ///   - completion: A completion closure that will return the image if it is successfully fetch either from cache or the network.
-  public func fetchImage(from url: URL, useCache: Bool = true, completion: @escaping Completion) {
+  public func fetchImage(from url: URL, configuration: Configuration = Configuration.default, completion: @escaping Completion) {
```

Comment:
@onmyway133 Oh this is pretty neat, this means that you can tailor the configuration per image view right?

Reply:
@zenangst yeah, we can customise per case, or use default one. We can even customise the cache as well

Reply:
Pretty darn nifty is if you ask me. Great work! 🚀

Reply:
Having the customization point as the call-site is really great